omod/src/main/webapp/resources/scripts/navigator/navigatorHandlers.js

```diff
@@ -144,7 +145,11 @@ function QuestionsKeyboardHandler(questionModels) {
             if(idx > 0) {
                 var previousIdx = findPreviousEnabledElement(idx, questions);
 
-                if (idx != previousIdx) {
+                  if (idx != previousIdx) {
```

Comment:
Looks like indentation disagrees within the `if(idx > 0)` block? But maybe it's just mixed tabs and spaces.

Reply:
fixed, thanks!